ChangeLog

```diff
@@ -0,0 +1,4 @@
+CHANGES
```

Comment:
I would drop the `ChangeLog` for the same reason. While there are valid arguments for a manually curated ChangeLog, in general we should be writing commit messages such that `git log` by itself generally provides the same information.

Reply:
done